### DISCUSSION TITLE
Medical Honeypots

### DISCUSSION THREAD

**Discussion by kriszdargo** [telekom-security/tpotce#1296] [Q&A]
Hello!

I would like to ask some help.
I tried to use the dicompot and the medpot in the standard and the medical mode too, but the medpot did not worked well and the dicompot either. 
They were running and healthy, but I could not reach the medpot, however I can reach the dicompot but only with one tool. I tried to reach them with telnet, ssh, http and they was not working. BUT the ports are open and the nmap shows the dicom and the hl7 services on the right ports. 

With the dicompot tried these:

- I tried to reach with the dcmtk: 
COMMAND: findscu -v -P -k "QueryRetrieveLevel=SERIES" <IP> <port>
RESULT:
I: Requesting Association
I: Association Accepted (Max Send PDV: 131060)
I: Sending Find Request (MsgID 1)
I: Request Identifiers:
I: 
I: # Dicom-Data-Set
I: # Used TransferSyntax: Little Endian Explicit
I: (0008,0052) CS [SERIES]                                 #   6, 1 QueryRetrieveLevel
I: 
I: Received Final Find Response (Success)
I: Releasing Association

But when I tried these command I got no result:
findscu -P -k PatientName="*" <IP> <PORT>
getscu -P -k PatientName="*" <IP> <PORT> 

I put these files in to the /data/dicompot/images directory as in the tpot.yml (and I add the volume too): https://wiki.cancerimagingarchive.net/pages/viewpage.action?pageId=80969777 . Maybe the newer version is the problem? But I can't find older here.

In the Medpot case:
- I can not reach the Medpot at all. ( I tried to reach it from the browser, from the terminal with curl, telnet, ssh, but it did not worked.)
How should I reach this pot? How can I test it like that?

Can you help me with this problems? In the Medpot case I absolutely have no idea whats wrong. 

Thank you in advance for your help!

**Comment by gtheodoridis** [comment]
I know is probably a bit late of response but in case you still need to know or somebody else is playing around with the dicompot, the problem with your commands is because of a typo, you should put a * inside of the quotes followed by the IP and the port.

findscu -P -k PatientName="*" IP PORT
getscu -P -k PatientName="*" IP PORT
